packages/syntax-core/src/CheckboxGroup/CheckboxGroup.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import { type StoryObj, type Meta } from "@storybook/react";
```

Comment:
@zprong can you make sure to run `pnpm run changeset` locally to add a changeset? This should be a minor change for `@cambly/syntax-core`